python/nx-cugraph/nx_cugraph/convert.py

```diff
@@ -60,6 +65,27 @@ def _iterate_values(graph, adj, is_dicts, func):
     return func(it), False
 
 
+# Consider adding this to `utils` if it is useful elsewhere
+def _fallback_decorator(func):
+    """Catch and convert exceptions to ``NotImplementedError``; use as a decorator.
+
+    ``nx.NetworkXError`` are raised without being converted. This allows
+    falling back to other backends if, for example, conversion to GPU failed.
+    """
+
+    @functools.wraps(func)
+    def inner(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except nx.NetworkXError:
+            raise
+        except Exception as exc:
+            raise _And_NotImplementedError(exc) from exc
```

Comment:
We should probably log something to prevent a bug from going unnoticed.

Reply:
The dispatch machinery ought to log or raise as appropriate.